recipes/pybroom/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pybroom" %}
+{% set version = "0.2" %}
+{% set sha256 = "45761b81e64556a8a649f4561cdc2f102a269aff114e23dc93081762724cd123" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tritemio/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
No need to template `name`, but if you are rolling out a bunch of personal packages like this I could see the value.